test-scripts/load-fixtures.js

```diff
@@ -2,80 +2,139 @@ const _ = require("lodash");
+    );
+  }
 }
 
-const patient = require("../test/fixtures/patients/patient-boris.json");
-const consentOrg = require("../test/fixtures/organizations/org-hl7.json");
-const actorOrg = require("../test/fixtures/organizations/org-good-health.json");
-const consent = require(CONSENT_FILE);
-
-patient.id = "thePatient";
-consentOrg.id = "theConsentOrg";
-actorOrg.id = "theActorOrg";
-consent.patient.reference = `Patient/${patient.id}`;
-consent.organization[0].reference = `Organization/${consentOrg.id}`;
-consent.provision.provision = consent.provision.provision.map(provision =>
-  _.set(
-    _.cloneDeep(provision),
-    "actor[0].reference.reference",
-    `Organization/${actorOrg.id}`
-  )
-);
-consent.dateTime = new Date().toISOString();
-
-const entries = [patient, actorOrg, consentOrg, consent].map(resource => ({
-  fullUrl: "",
-  resource: resource,
-  request: {
-    method: "PUT",
-    url: `${
-      resource.resourceType
-    }?identifier=${LEAP_IDENTIFIER_SYSTEM}|${getLeapIdentifer(resource)}`
+async function removeResourcesByTypes(resourceTypes) {
+  for (const resourceType of resourceTypes) {
+    const ids = await getAllResourceIds(resourceType);
+    await removeAllResourcesByTypeAndIds(resourceType, ids);
   }
-}));
-
-const transaction = {
-  resourceType: "Bundle",
-  id: "bundle-transaction",
-  meta: {
-    lastUpdated: new Date().toISOString()
-  },
-  type: "transaction",
-  entry: entries
-};
-
-superagent
-  .post(FHIR_BASE)
-  .set("Content-Type", "application/json")
-  .send(transaction)
-  .then(res => {
-    console.log(JSON.stringify(res.body, null, 2));
-  })
-  .catch(err => {
-    console.log(err);
-  });
+}
+
+function prepareEntries() {
+  const consent = require(`${PATH}/consents/r5/${CONSENT_FILE}`);
+  const patient = require(`${PATH}/patients/patient-boris.json`);
+  const consentOrg = require(`${PATH}/organizations/org-hl7.json`);
+  const actorOrg = require(`${PATH}/organizations/org-good-health.json`);
+
+  patient.id = "thePatient";
+  consentOrg.id = "theConsentOrg";
+  actorOrg.id = "theActorOrg";
+
+  consent?.subject?.reference &&
+    (consent.subject.reference = `Patient/${patient.id}`);
+  consent?.manager?.[0]?.reference &&
+    (consent.manager[0].reference = `Organization/${consentOrg.id}`);
+  consent?.provision &&
+    (consent.provision = consent.provision.map((provision) =>
+      _.set(
+        _.cloneDeep(provision),
+        "actor[0].reference.reference",
+        `Organization/${actorOrg.id}`
+      )
+    ));
+  consent.dateTime && (consent.dateTime = new Date().toISOString());
+
+  return [patient, actorOrg, consentOrg, consent].map((resource) => ({
+    fullUrl: `urn:uuid:${LEAP_IDENTIFIER_SYSTEM}:${getLeapIdentifer(resource)}`,
+    resource: resource,
+    request: {
+      method: "PUT",
+      url: `${
+        resource.resourceType
+      }?identifier=${LEAP_IDENTIFIER_SYSTEM}|${getLeapIdentifer(resource)}`
+    }
+  }));
+}
+
+async function populateHapiDatabase() {
```

Comment:
this is not a good name. this is not pushing to the database directly and is not specific to HAPI.